tests/posix/common/src/mqueue.c

```diff
@@ -97,4 +96,205 @@ ZTEST(mqueue, test_mqueue)
 	zassert_false(mq_unlink(queue), "Not able to unlink Queue");
 }
 
+bool notification_executed;
```

Comment:
Not blocking either, so only if a respin is needed.
```suggestion
static bool notification_executed;
```